libp2p/NodeTable.h

```diff
@@ -183,21 +250,24 @@ class NodeTable: UDPSocketEvents, public std::enable_shared_from_this<NodeTable>
 	/// Sends FindNeighbor packet. See doFindNode.
 	void requestNeighbours(NodeEntry const& _node, NodeId _target) const;
 
+	std::unique_ptr<NodeTableEventHandler> m_nodeEvents;		///< Event handler for node events.
```

Comment:
From the name it looks like a list of events, but it actually is an event handler.